ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/StructKeyUtil.java

```diff
@@ -58,7 +58,7 @@ public static KeyBuilder keyBuilder(final ColumnName name, final SqlType type) {
   }
 
   @SuppressWarnings("unchecked")
-  public static List<?> asList(final Object key) {
+  public static List<Object> asList(final Object key) {
```

Comment:
Is there a reason this was previously `List<?>` rather than `List<Object>`? I don't fully understand the meaning of this change, but it lets us avoid an unchecked cast downstream.